core/include/drivers/imx_i2c.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2020 Foundries.io
+ * Jorge Ramirez-Ortiz <jorge@foundries.io>
+ */
+#ifndef __DRIVERS_IMX_I2C_H
+#define __DRIVERS_IMX_I2C_H
+
+#include <stdint.h>
+#include <tee_api_types.h>
+
+TEE_Result imx_i2c_write(uint8_t bid, uint8_t chip, const uint8_t *p, int l);
```

Comment:
Maybe rename `chip` to `chip_addr`, or `i2c_addr`,  or something highlighting it is the I2C address of the slave.
Or maybe `chip` is often used as an alias for the I2C address of slaves in I2C literature.

Reply:
yes, in i2c semantics chip  translates to the slave and the only way to access it is via its address...I'd rather keep it as is.

Reply:
ok, fine.